x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule.scss

```
@@ -14,3 +14,7 @@
   bottom: $euiSizeXS;
   position: relative;
 }
+
+.euiEmptyPrompt--vertical .euiEmptyPrompt__main {
+ min-height: unset;
+}
```

Please don't do this. This is now a universal selector that can be applied to any `EuiEmptyPrompt` on the page. You **must** scope your selectors by creating a custom class name for the EUI component first.

```suggestion
.customEmptyPromptClass .euiEmptyPrompt__main {
 min-height: unset;
}
```

---

I just jumped into this PR to assist, and will tidy this up/change it.

---

It is removed!!!!